src/main/java/at/hannibal2/skyhanni/features/inventory/chocolatefactory/ChocolateFactoryBarnManager.kt

```diff
@@ -18,6 +18,7 @@ import kotlin.time.Duration.Companion.seconds
 object ChocolateFactoryBarnManager {
 
     private val config get() = ChocolateFactoryAPI.config
+    private val eventConfig get() = HoppityEggsManager.config
```

Comment:
why is this called event?

Reply:
Fixed in [689d42b](https://github.com/hannibal002/SkyHanni/pull/1887/commits/689d42bbd824c85181237c50d7003f36fca5a8ca)